tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -341,14 +340,15 @@ def _execute_test_suite(self, suite: TestSuiteInfo) -> bool:
         suite_full_name = f"{suite_name}-{self.context.image_name}"
         suite_start_time: datetime.datetime = datetime.datetime.now()
 
+        success: bool = True  # True if all the tests succeed
```

Comment:
This method is supposed to return True on success; it was returning True on failure